canonical-web-and-design/ubuntu.com
# change "Canonical" nav item to "Company"

## Done

- The new dropdown nav item added to the top nav mistakenly read "Canonical", based on old design. This PR updates it to "Company".

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- See that there's an item in the top nav that reads "Company", and the dropdown works
